recipes/util-linux/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "util-linux" %}
+{% set version = "2.30" %}
+{% set md5 = "90e0722c82767c98861f608f9845345b" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://www.kernel.org/pub/linux/utils/util-linux/v2.30/util-linux-2.30.tar.gz
+    md5: {{ md5 }}
```

Comment:
Could we please use a better checksum than `md5`? Ideally `sha256`. Though if they provide one that is better than `md5` that would be worth going with.